src/components/v5/shared/Stepper/partials/StepperButton/types.ts

```diff
@@ -0,0 +1,12 @@
+import { ButtonHTMLAttributes } from 'react';
+import { TooltipProps } from '~shared/Extensions/Tooltip/types';
+
+export interface StepperButtonProps
+  extends ButtonHTMLAttributes<HTMLButtonElement> {
+  label: string;
+  stage: 'completed' | 'current' | 'upcoming' | 'skipped';
```

Comment:
Maybe it would be better to keep it in const? And then generate a type from it like here: https://github.com/JoinColony/colonyCDapp/blob/5643298d0a691713f23adcf729efdebe79e27dcd/src/components/v5/common/Fields/consts.ts#L1